### DISCUSSION TITLE
Is there a solution to increase size

### DISCUSSION THREAD

**Discussion by ishtiaqSamdani** [pmndrs/react-spring#1885] [Support]
is there any solution to increase the size of the parallax layer component?

**Comment by joshuaellis** [comment]
I'm not sure what you're asking, do you have a demo of what you're aiming to achieve?

**Comment by kindoflew** [comment]
The size along the scrolling axis can be increased with the `factor` prop. Alternatively, you could add a `style` prop to it and style it like any other `div`. just know that certain styles might override ones that we use internally for the parallax effect: https://github.com/pmndrs/react-spring/blob/28ef2d1bb5ef993ec8fbe156399bf27d4093bbce/packages/parallax/src/index.tsx#L185
